youtube_dl/extractor/atresplayer.py

```diff
@@ -79,124 +64,83 @@ def _login(self):
             return
 
         login_form = {
-            'j_username': username,
-            'j_password': password,
+            'username': username,
+            'password': password,
         }
 
+        self._download_webpage(self._LOGIN_URL, None, 'get login page')
         request = sanitized_Request(
```

Comment:
Inline into actual `_download_*` call.

Reply:
I don0t understand this comment.

Reply:
@Nekmo You can make json POST with `_download_json`, and set expected statuses (`data=urlencode_postdata(form_data)`)

From common.py
``` python
      def _download_json(
            self, url_or_request, video_id, note='Downloading JSON metadata',
            errnote='Unable to download JSON metadata', transform_source=None,
            fatal=True, encoding=None, data=None, headers={}, query={},
            expected_status=None):
        """
        Return the JSON object as a dict.

        See _download_webpage docstring for arguments specification.
        """
```

Reply:
The response from the server is not a json. This request is to set cookies.